interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -147,57 +147,100 @@ components:
-        - startDateTime
-        - endDateTime
+        - checks
       properties:
-        position:
-          $ref: '#/components/schemas/Position'
-        uSpaceClass:
-          $ref: '#/components/schemas/USpaceClass'
-        acceptableRestrictions:
+        checks:
           type: array
           items:
-            $ref: '#/components/schemas/Restriction'
-        startDateTime:
+            $ref: '#/components/schemas/GeozonesCheck'
+
+    GeozonesCheck:
+      type: object
+      required:
+        - filterSets
+      properties:
+        filterSets:
+          description: >-
+            Select Geozones which match any of the specified filter sets.
+          type: array
+          items:
+            $ref: '#/components/schemas/GeozonesFilterSet'
+
+    GeozonesFilterSet:
+      description: >-
+        Set of filters to select only a subset of Geozones.  Only Geozones which are applicable to all specified filters within this filter set should be selected.
+      type: object
+      properties:
+        position:
+          description: >-
+            If specified, only select Geozones encompassing this position.
+          anyOf:
+            - $ref: '#/components/schemas/Position'
+        after:
+          description: >-
+            If specified, only select Geozones which are defined at or after this time.
           type: string
           format: date-time
-        endDateTime:
+        before:
+          description: >-
+            If specified, only select Geozones which are defined at or before this time.
           type: string
           format: date-time
+        ed269:
+          $ref: '#/components/schemas/ED269Filters'
+
+    ED269Filters:
+      description: >-
+        Filter criteria for the selection of Geozones according to ED-269 characteristics.
+      type: object
+      properties:
+        uSpaceClass:
+          description: >-
+            If specified, only select Geozones which are of the specified `uSpaceClass`.
+          anyOf:
+            - $ref: '#/components/schemas/USpaceClass'
+        acceptableRestrictions:
+          description: >-
+            If specified and non-empty, only select Geozones which are one of the specified restriction types.
+          type: array
+          items:
+            $ref: '#/components/schemas/Restriction'
 
-    UASZonesCheckReply:
+    GeozonesCheckReply:
       type: object
       properties:
-        applicableUASZone:
+        applicableGeozone:
           type: boolean
           description: >-
-            True if there is one or more applicable UASZone under the requested conditions.
-        error:
-          description: Human-readable explanation of the result for debugging purpose only. This field is required when the check do not complete.
+            True if and only if one or more applicable Geozones were selected according to the specified selection criteria.
+          default: false
 
 paths:
-  /v1/status:
+  /status:
```

Comment:
Although other APIs include a version prefix like this, I think we shouldn't do that going forward.  USSPs are free to make the base URL whatever they want, so mandating a shared prefix like this is unnecessary and overly-constraining.  I suspect a common desire would be to include the API version in the base URL, and that API version is a full semantic version (like 0.1.0) rather than simply "v1" (there are multiple "v1"s).

Reply:
Sounds good to me.